test/core/avl/BUILD

```diff
@@ -27,4 +27,5 @@ grpc_cc_test(
         "//:grpc",
         "//test/core/util:grpc_test_util",
     ],
+    uses_polling = False,
```

Comment:
this is a proof of concept: Later I'll need to do this for all tests that set uses_polling: false in build.yaml